core-bundle/src/Util/PackageUtil.php

```diff
@@ -21,16 +21,21 @@ class PackageUtil
      */
     public static function getVersion(string $packageName): string
     {
-        $version = Versions::getVersion($packageName);
+        $version = InstalledVersions::getPrettyVersion($packageName) ?? '';
 
-        return static::parseVersion($version);
+        return ltrim($version, 'v');
     }
 
     /**
      * Returns the version number as "major.minor.patch".
+     *
+     * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0;
+     *             use the getVersion() method instead
      */
     public static function getNormalizedVersion(string $packageName): string
     {
+        trigger_deprecation('contao/core-bundle', '4.13', 'The PackageUtil::getNormalizedVersion() method has been deprecated and will no longer work in Contao 5.0. Use the PackageUtil::getVersion() method instead.');
```

Comment:
This should not be deprecated but instead use `getVersion()` of Composer's `InstalledVersions`. This is the one you should use for `version_compare()` calls.

Reply:
If you read the phpDoc comment, the method is supposed to return the version as `major.minor.patch`, which is not what `InstalledVersions::getVersion()` does. We have never used the method in the core and I don't want to maintain it in the future, therefore I have deprecated it.